op-node/node/server_test.go

```diff
@@ -138,7 +138,7 @@ func TestVersion(t *testing.T) {
 		// ignore other rollup config info in this test
 	}
 	m := &opmetrics.NoopRPCMetrics{}
-	server := newRPCServer(rpcCfg, rollupCfg, l2Client, drClient, safeReader, log, m, "0.0")
+	server := newRPCServer(rpcCfg, rollupCfg, nil, l2Client, drClient, safeReader, log, m, "0.0")
```

Comment:
Should we be adding a new test here to get the dependency set?

Reply:
Yeah good call - done.